### PR TITLE
Serbian anthem

### PR DESCRIPTION
**Pull request by bokimitic**
anthem

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab can you review this PR please?

File: serbian/drugasrpska/himna.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+Српска химна
```
Comment (casey-collab) [inline serbian/drugasrpska/himna.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by bokimitic** [comment]
Now it's ready to merge.

**Pull request closed without merging**